request-processor-celery/tests/integration/test_tasks.py

```diff
@@ -1,25 +1,142 @@
 import datetime
 import time
-
+from collections import namedtuple
 import pytest
-
+import csv
+import json
+import shutil
+import os
 import database
 from request_model import models, schemas
 from tasks import check_datafile
 
 
-def test_check_datafile(celery_app, celery_worker, s3_bucket, db):
+@pytest.fixture
```

Comment:
Let's move this fixture to `conftest.py` to keep this file cleaner and more focussed.